bs/__tests__/tablecloth_test.ml

```diff
@@ -17,7 +17,9 @@ let () =
     test "indexedMap two elements" (fun () -> expect (List.indexedMap ~f:(fun i _ -> i) ['a';'b']) |> toEqual [0;1]);
 
     test "init empty list" (fun () -> expect (List.init []) |> toEqual None);
+    (* This seems kinda wrong, if this is correct, delete the comments, proposed solution in tablecloth.ml *)
     test "init one element" (fun () -> expect (List.init ['a']) |> toEqual (Some []));
```

Comment:
`List.init` is like `List.tail`, so whatever List.tail does, List.init should do too.

`List.tail []` => `None`
`List.tail [1]` => []`
`List.tail [1;2]` => [2]`

`List.init []` => `None`
`List.init [1]` => []`
`List.init [1;2]` => [1]`

Reply:
thats what i thought so there was a bug

Reply:
ah no, everything worked as intended